extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -21,18 +21,20 @@ describes.realWin(
     let pageElement;
 
     const defaultInlineConfig = {
-      items: [
+      'items': [
         {
-          'productId': 'city-pop',
-          'productTitle': 'Plastic Love',
-          'productPrice': 19,
+          'productId': 'art',
```

Comment:
This object needs values different from `examples/amp-story/shopping/remote.json`.